src/components/TodoList/index.jsx

```diff
@@ -1,13 +1,15 @@
-import React from 'react';
-import TodoItem from '../TodoItem';
+import React from "react";
+import { connect } from "react-redux";
+import TodoItem from "../TodoItem";
 
-function TodoList() {
-  return (
-    <div>
-      <TodoItem />
-      <TodoItem />
-    </div>
-  );
+function TodoList({ todos }) {
+  const renderTodos = todos.map((todo) => (
```

Comment:
Render components directly inside the return statement instead of storing it in a variable. This has performance complications as `renderTodos` is treated as a computed variable which is re-rendered each time this component or any of its parents change instead of only rendering the changed UI if anything changed.